web/index.html

```
@@ -4,10 +4,11 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta name="description" content="SynthOrg -- monitoring and management dashboard for synthetic organizations" />
-    <!-- Per-request CSP nonce. Substituted by nginx sub_filter at serve time;
-         read at runtime by lib/csp.ts and propagated to CSPProvider (Base UI)
-         and MotionConfig (Framer Motion) for inline <style> tag signing. -->
-    <meta name="csp-nonce" content="__CSP_NONCE__" />
+    <!-- Per-request CSP nonce. Substituted by Caddy templates directive at
+         serve time; read at runtime by lib/csp.ts and propagated to
+         CSPProvider (Base UI) and MotionConfig (Framer Motion) for inline
+         <style> tag signing. -->
+    <meta name="csp-nonce" content="{{placeholder "http.request.uuid"}}" />
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)